### ISSUE TITLE
Port forwarding config error

### ISSUE THREAD

**Issue by jakejscott** [flynn-archive/flynn-demo#34]
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'flynn-base'...
==> default: Matching MAC address for NAT networking...
==> default: Setting the name of the VM: flynn-demo_default_1399272875503_63510
Vagrant cannot forward the specified ports on this VM, since they
would collide with some other application that is already listening
on these ports. The forwarded port to 8080 is already in use
on the host machine.

To fix this, modify your current projects Vagrantfile to use another
port. Example, where '1234' would be replaced by a unique host port:

  config.vm.network :forwarded_port, guest: 80, host: 1234

Sometimes, Vagrant will attempt to auto-correct this for you. In this
case, Vagrant was unable to. This is usually because the guest machine
is in a state which doesn't allow modifying port forwarding.

**Comment by jayd3e** [comment]
Are you running an application on your machine that uses port 8080?

**Comment by jayd3e** [comment]
Either way, the solution is pretty cut and dry.  If you read the error message, you need to either change the port in the Vagrantfile to something other than 8080(which will require you to change some default valued when running certain Flynn commands).  Or you need to shutdown anything that is using port 8080 on your host machine(the machine running the VM).  I would recommend the later.

**Comment by jakejscott** [comment]
Okay, sorry for being a muppet :)

I'm now running into another issue:

==> default: Importing base box 'flynn-base'...
==> default: Matching MAC address for NAT networking...
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
==> default: Forwarding ports...
    default: 80 => 8080 (adapter 1)
    default: 443 => 8081 (adapter 1)
    default: 2222 => 2201 (adapter 1)
    default: 22 => 2222 (adapter 1)
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
    default: Warning: Connection timeout. Retrying...
    default: Warning: Remote connection disconnect. Retrying...
    default: Warning: Remote connection disconnect. Retrying...
    default: Warning: Remote connection disconnect. Retrying...

**Comment by jayd3e** [comment]
Let that run, it should eventually connect.  Its just attempting to connect via SSH while the machine is booting.

**Comment by jakejscott** [comment]
After a couple of minutes it times out and I get a vagrant error.

**Comment by jayd3e** [comment]
Maybe try a `vagrant reload`.  That usually happens when the guest machine is waiting on input, or when its doing startup checks of some sort.
